src/ECER.Clients.RegistryPortal/ecer.clients.registryportal.client/src/components/Message.vue

```diff
@@ -39,20 +32,13 @@
     </v-dialog>
   </div>
   <div v-if="mdAndUp && messageStore.currentMessage != null">
-    <v-btn
-      v-if="messageStore.currentMessage?.doNotReply == false"
-      prepend-icon="mdi-reply"
-      variant="text"
-      color="primary"
-      text="Reply"
-      @click="handleMessageReply"
-    >
-      Reply
-    </v-btn>
-    <h2 class="mt-6">{{ messageStore.currentMessage?.subject }}</h2>
+    <v-sheet v-if="messageStore.currentMessage?.doNotReply == false" class="message-reply mb-6">
```

Comment:
If we don't think we will need to use these classes in other areas, I would suggest just putting these in the style tag within the component. 

like 
<style scoped> <--- make sure it's scoped so it doesn't leak 
.message-reply {
  what you need here
}
</style>

this way we have less global styles if we're just using it in one place.

Reply:
then we don't need to add global styles in the .scss files.

Reply:
Agreed, Done.